src/download.rs

```diff
@@ -401,6 +463,25 @@ mod error {
     #[derive(Debug, Snafu)]
     #[snafu(visibility = "pub(super)")]
     pub(super) enum Error {
+        #[snafu(display("Failed to read metadata for '{}': {}", path.display(), source))]
+        ReadFileMetadata {
+            path: PathBuf,
+            source: std::io::Error,
+        },
+
+        #[snafu(display("Failed to get block device size: {}", source))]
+        GetBlockDeviceSize { source: nix::Error },
+
+        #[snafu(display("Got invalid block device size: {}", result))]
```

Comment:
Nit: I'd remove the word "Got"

Reply:
Thanks Zac, for reviewing.  I have made the change.